src/common/darktable.h

```diff
@@ -279,6 +279,23 @@ typedef struct dt_codepath_t
   unsigned int OPENMP_SIMD : 1; // always stays the last one
 } dt_codepath_t;
 
+typedef enum dt_sys_resources_levels_t
```

Comment:
we should get rid of this typedef/struct otherwise we need to synchronize with darktablerc.xml.in. The value must be read where needed using the conf api. This will avoid also the need to restart dt.

Reply:
Sure - did remove that.

BTW - restarting dt is only required for mipmap cache changes as re-initialize that seems to be unsupported or at least very errorprone.